paml_convert/markdown/excel_to_experimentdata.py

```diff
@@ -12,7 +12,7 @@
 
 doc = paml.Document()
 
-wb = openpyxl.load_workbook(filename ='../test/testfiles/BBN_LUDOX_OD_calibration_2021-3.xlsx')
+wb = openpyxl.load_workbook(filename ='../../test/testfiles/BBN_LUDOX_OD_calibration_2021-3.xlsx')
 ws = wb['Data Reporting']
```

Comment:
It looks like this belongs in testing and/or demos rather than a code directory.